coldfront/core/allocation/tasks.py

```diff
@@ -41,148 +45,174 @@ def update_statuses():
-                            'email/allocation_expired.txt',
-                            template_context,
-                            EMAIL_SENDER,
-                            email_receiver_list
-                            )
-
-        logger.info('Allocation to {} expired email sent to PI {}.'.format(
-            resource_name, allocation_obj.project.pi.username))
+        
+        expring_in_days = (datetime.datetime.today() + datetime.timedelta(days=-1)).date()
+                
+        for allocationuser in user.allocationuser_set.all():
+            allocation = allocationuser.allocation
+
+            if (allocation.end_date == expring_in_days):
+                
+                project_url = f'{CENTER_BASE_URL.strip("/")}/{"project"}/{allocation.project.pk}/'
+
+                allocation_renew_url = f'{CENTER_BASE_URL.strip("/")}/{"allocation"}/{allocation.pk}/{"renew"}/'
+
+                resource_name = allocation.get_parent_resource.name
+
+                template_context = {
+                    'center_name': CENTER_NAME,
+                    'project_dict': projectdict,
+                    'allocation_dict': allocationdict,
+                    'project_renewal_help_url': CENTER_PROJECT_RENEWAL_HELP_URL,
+                    'opt_out_instruction_url': EMAIL_OPT_OUT_INSTRUCTION_URL,
+                    'signature': EMAIL_SIGNATURE
+                }
+
+                expire_notification = allocation.allocationattribute_set.filter(
+                    allocation_attribute_type__name='EXPIRE NOTIFICATION').first()
+                if expire_notification and expire_notification.value == 'No':
+                    continue
+
+                for projectuser in allocation.project.projectuser_set.filter(user=user, status__name='Active'): 
+                    if ((projectuser.enable_notifications) and 
+                        (allocationuser.user == user and allocationuser.status.name == 'Active')):
+
+                        if (user.email not in email_receiver_list):
+                            email_receiver_list.append(user.email)
+
+                        if project_url not in allocationdict:
+                                allocationdict[project_url] = []
+                                allocationdict[project_url].append({allocation_renew_url : resource_name})
+                        else:
+                            if {allocation_renew_url : resource_name} not in allocationdict[project_url]:
+                                allocationdict[project_url].append({allocation_renew_url : resource_name})
+
+                        if allocation.project.title not in projectdict:
+                            projectdict[allocation.project.title] = (project_url, allocation.project.pi.username)
+
+                        admin_expire_notification = allocation.allocationattribute_set.filter(
+                            allocation_attribute_type__name='ADMIN EXPIRE NOTIFICATION').first()
```

Comment:
The name `ADMIN EXPIRE NOTIFICATION` is confusing and the fact that it's presence means to NOT do something counter intuitive. Let's tweak this a bit:

1. Remove `ADMIN EXPIRE NOTIFICATION` allocation attribute
2. Add a global config setting ]EMAIL_ADMINS_ON_ALLOCATION_EXPIRE` and make this false by default

This will enable center admins to enable this setting and will get an email any time an allocation expires.